chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -111,7 +111,11 @@ attachmentBasePath.clearBasePath.button                        = Clear Base Dire
 dataDir.notFound						= The Zotero data directory could not be found.
 dataDir.previousDir					= Previous directory:
 dataDir.useProfileDir				= Use %S profile directory
+dataDir.existingDirInDropbox.title  = Data Directory In Dropbox
+dataDir.existingDirInDropbox.text   = Your current data directory appears to be in a Dropbox directory. This might lead to data corruption.\n\nChange Zotero data directory now?
 dataDir.selectDir					= Select a Zotero data directory
+dataDir.selectedDirInDropbox.title		= Directory In Dropbox
+dataDir.selectedDirInDropbox.text		= The directory you selected appears to be in a Dropbox directory. This might lead to data corruption.\n\nCreate Zotero files in this directory anyway?
```

Comment:
This is accurate, but we can probably just assume that it's Dropbox. (If someone has a path with "dropbox" in it that's not Dropbox, they'll probably understand, in any case.)

How about: "Choosing a data directory within Dropbox may corrupt your Zotero database.\n\nUse this directory anyway?"